libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -1,66 +1,61 @@
 // (C) 2021 GoodData Corporation
-
 import * as Navigation from "../../tools/navigation";
 import { Table } from "../../tools/table";
 import { Widget } from "../../tools/widget";
 import { DrillToModal } from "../../tools/drillToModal";
-import { getBackend, getHost, getProjectId } from "../../support/constants";
-
+import { DateFilter } from "../../tools/dateFilter";
+import { DateFilterValue } from "../../tools/enum/DateFilterValue";
+import { Api } from "../../tools/api";
 //!!!!!!!!!!! this constant could be shifted check gray pages if test failed before each or after hook
 const DEPARTMENT_ID = "1090";
 //!!!!!!!!!!! this constant could be shifted check gray pages if test failed before each or after hook
-const PRODUCT_ID = "1056";
+const PRODUCT_ID = "1057";
+const YEAR_CLOSE = "503";
+const QUARTER_CLOSE = "522";
 const drillModal = new DrillToModal();
+const api = new Api();
+const tableData = [
+    {
+        title: "table has row column measure",
+        values: ["CompuSci", "$4,163,871.80", "$11,418,823.89"],
+    },
+    {
+        title: "table only row and column",
+        values: ["CompuSci"],
+    },
+    {
+        title: "table only rows and measures",
+        values: ["CompuSci", "$15,582,695.69"],
+    },
+    {
+        title: "table only rows",
+        values: ["CompuSci"],
+    },
+];
+const firstDrillValue = ["617.00", "1,903.00", "3,526.00", "2,046.00", "41.00", "220.10", "386.70", "281.00"];
```

Comment:
which constant only used in scope locally, then define it in the test, don't define it globally